.travis.yml

```diff
@@ -3,6 +3,15 @@ scala:
   - 2.11.8
 jdk:
   - oraclejdk8
+# These directories are cached to S3 at the end of the build
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
```

Comment:
I did some benchmarks doing precisely that some months ago but didn't notice any difference in build time (I don't remember if it was for OCS or OCS3 though). In any case this is very useful when there are download timeouts 👍 .